src/Arcus.Testing.Tests.Integration/Storage/Fixture/NoSqlTestContext.cs

```diff
@@ -163,12 +163,9 @@ public async Task ShouldStoreContainerAsync(string containerId)
         /// </summary>
         public async Task ShouldNotStoreContainerAsync(string containerId)
         {
-            await Poll.UntilAvailableAsync<XunitException>(async () =>
```

Comment:
By removing the poll, we test that the container is indeed immediately removed after the disposal.